src/particles/collisions/ElasticCollisionPerez.H

```diff
@@ -53,25 +56,25 @@ void ElasticCollisionPerez (
     T_R const  q1, T_R const  q2,
     T_R const  m1, T_R const  m2,
     T_R const  T1, T_R const  T2,
-    T_R const  dt, T_R const   L, T_R const dV,
+    T_R const  dt, T_R const   L, T_R const dV, const PhysConst& cst, const bool normalized_units,
+    const amrex::Real background_density_SI,
     amrex::RandomEngine const& engine)
 {
     using namespace amrex::literals;
-
-    T_R constexpr inv_c2 = T_R(1.0)/(PhysConstSI::c*PhysConstSI::c);
+    T_R const inv_c2 = T_R(1.0)/(cst.c*cst.c);
     const int NI1 = I1e - I1s;
     const int NI2 = I2e - I2s;
 
     // get local T1t and T2t
     T_R T1t; T_R T2t;
     if ( T1 <= T_R(0.0) && L <= T_R(0.0) )
     {
-        T1t = ComputeTemperature(I1s,I1e,I1,u1x,u1y,psi1,m1);
+        T1t = ComputeTemperature(I1s,I1e,I1,u1x,u1y,psi1,m1, cst);
```

Comment:
Just to be consistent within this line
```suggestion
        T1t = ComputeTemperature(I1s,I1e,I1,u1x,u1y,psi1,m1,cst);
```